### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by lisafrench**
This PR:
- updates support documentation
- makes headline capitalization consistent
- makes headlines match sidebar outline headlines
- fixes some spelling errors
- adds a page for on-boarding
- adds info on our development cycle process
- renames file called `github` to be `development_process` to more accurately reflect the file content

Note:
- some formatting on save changes have been made
- the main things to review are the new `onboarding` file and the added development process documentation located in the `development_process` file

### CODE REVIEW

File: docs/growth.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
-# Growth and Challenge
+# Growth
 
 Growth in our careers is directly proportional to the scale of problems
 we can effectively solve. To grow, we must advocate for ever-larger problems
 and continually learn how to approach and solve them.
 
 Stratasan has a high level of commitment to support our employees' ongoing
 education and career growth. However, actually making use of this is our own
-indvidual responsibility. This may include spending personal time on learning
```
Comment (sburns) [inline docs/growth.md:9]: So you're saying I misspelled individual differently in the same document. Sounds about right :/

**Review by sburns — APPROVED** [review]
I think these changes look great!

**Comment by lisafrench** [comment]
@sburns Thanks for reviewing! Should I worry about these checks passing before merging?

**Comment by sburns** [comment]
I need to tweak the GitHub actions on this branch :(

**Pull request merged**